plus/src/test/java/org/apache/calcite/adapter/os/OsAdapterTest.java

```diff
@@ -83,9 +88,22 @@ private static boolean hasGit() {
     }
   }
 
+  private static boolean checkProcessExists(String command) {
+    try {
+      Process process = new ProcessBuilder().command(command).start();
+      Assert.assertNotNull(process);
+      int errCode = process.waitFor();
+      Assert.assertEquals(0, errCode);
+      return true;
+    } catch (AssertionError | IOException | InterruptedException e) {
+      return false;
+    }
+  }
+
   @Test public void testDu() {
     Assume.assumeFalse("Skip: the 'du' table does not work on Windows",
         isWindows());
+    Assume.assumeTrue(checkProcessExists("du"));
```

Comment:
Would you please use `assumeTrue(String message, boolean b)`, so failed assumptions have corresponding messages?

E.g. `assumeToolExists("du")`

Reply:
Ah yea good call.